noxfile.py

```diff
@@ -45,6 +45,14 @@
 @nox.session(python=["3.6", "3.7", "3.8", "3.9", "3.10", "3.11"])  # type: ignore
 def test(session: Any) -> None:
     session.install(".")
+    # ensure client can be imported without aiohttp
+    session.run("python", "-c", "import opensearchpy\nprint(opensearchpy.OpenSearch())")
```

Comment:
This reproduces #592 without the fix.

Reply:
I have full faith that this should work, but looking through the tests I don't think we do a `nox test` in any of the CIs. I think we really need to add that.